src/equations/equations_parabolic.jl

```diff
@@ -1,3 +1,11 @@
+# specify transformation of conservative variables prior to taking gradients.
+# specialize this function to compute gradients e.g., of primitive variables instead of conservative
+gradient_variable_transformation(::AbstractEquationsParabolic, dg_parabolic) = cons2cons
+
 # Linear scalar diffusion for use in linear scalar advection-diffusion problems
 abstract type AbstractLaplaceDiffusionEquations{NDIMS, NVARS} <: AbstractEquationsParabolic{NDIMS, NVARS} end
 include("laplace_diffusion_2d.jl")
+
+# Compressible Navier-Stokes equations
+abstract type AbstractCompressibleNavierStokesEquations{NDIMS, NVARS} <: AbstractEquationsParabolic{NDIMS, NVARS} end
```

Comment:
If we rename the specific equations (see comment of @sloede above), we should also rename the abstract equations.

Reply:
Doing so now. Also renamed `AbstractLaplaceDiffusionEquations`